src/styles/main.scss

```diff
@@ -1,3 +1,20 @@
-body {
-  margin: 0;
-}
+@import url('https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@400;700&family=Space+Mono&display=swap');
```

Comment:
Consider locally hosting the fonts instead of using a third party url. This can improve loading time and performance.